### ISSUE TITLE
domm-acario-light sets wrong comment font.

### ISSUE THREAD

**Issue by saintaugustinus** [hlissner/emacs-doom-themes#423]
Hi, I'm using domm-acario-light theme right now, and I found it set wrong font in comment.
Here it is:
Beside the comment the font is  Iosevka:
<img width="1051" alt="截屏2020-03-20下午7 02 17" src="https://user-images.githubusercontent.com/3031225/77158497-65b70a80-6ade-11ea-9b7d-e359f380ed51.png">

The comment font is Courier:
<img width="1051" alt="截屏2020-03-20下午7 02 05" src="https://user-images.githubusercontent.com/3031225/77158534-74052680-6ade-11ea-8778-4a02ed03d8f4.png">

And I found this pull request: #356 

I try to revert it and the comment font get right,

```
(font-lock-comment-face
    :inherit 'fixed-pitch-serif ;; I try to delete this line the comment font get right two
    :slant 'italic
    :foreground comments
    :background (if doom-acario-light-comment-bg (doom-darken bg 0.05)))
```

Is this a bug, and how to fix it ?

**Comment by ema2159** [comment]
@gagbo

**Comment by gagbo** [comment]
It's down to taste at this point, I'll try to make this optional then :)

**Comment by gagbo** [comment]
Quick workaround is to set the `fixed-pitch-serif` font to iosevka :

- Using doom : just `(setq doom-serif-font (font-spec :family "Iosevka"))` in your config.el
- Not using doom : `(set-face-attribute 'fixed-pitch-serif nil :family "Iosevka")` or `(set-face-attribute 'fixed-pitch-serif nil :font (font-spec :family "Iosevka"))` should work

**Comment by gagbo** [comment]
( FTR solved by a008d1158398514a133c5a45562af007c77bd7f7 )
